cypress/e2e/marc-authority/marc-authority-edit.cy.js

```diff
@@ -0,0 +1,71 @@
+import getRandomPostfix from '../../support/utils/stringTools';
+import TestTypes from '../../support/dictionary/testTypes';
+import DevTeams from '../../support/dictionary/devTeams';
+import Permissions from '../../support/dictionary/permissions';
+import TopMenu from '../../support/fragments/topMenu';
+import DataImport from '../../support/fragments/data_import/dataImport';
+import MarcAuthority from '../../support/fragments/marcAuthority/marcAuthority';
+import Users from '../../support/fragments/users/users';
+import JobProfiles from '../../support/fragments/data_import/job_profiles/jobProfiles';
+import Logs from '../../support/fragments/data_import/logs/logs';
+import MarcAuthorities from '../../support/fragments/marcAuthority/marcAuthorities';
+import QuickMarcEditor from '../../support/fragments/quickMarcEditor';
+
+describe('MARC Authority -> Edit Authority record', () => {
+  const testData = {
+    authority: {
+      title: 'Twain, Mark, 1835-1910. Adventures of Huckleberry Finn',
+      searchOption: 'Keyword',
+    }
+  };
+  const jobProfileToRun = 'Default - Create SRS MARC Authority';
+  const fileName = `testMarcFile.${getRandomPostfix()}.mrc`;
+  let createdAuthorityID;
+
+  before('', () => {
+    cy.createTempUser([
+      Permissions.uiMarcAuthoritiesAuthorityRecordEdit.gui,
+      Permissions.uiMarcAuthoritiesAuthorityRecordView.gui,
+      Permissions.uiQuickMarcQuickMarcAuthoritiesEditorAll.gui,
+    ]).then(createdUserProperties => {
+      testData.userProperties = createdUserProperties;
+    });
+
+    cy.loginAsAdmin({ path: TopMenu.dataImportPath, waiter: DataImport.waitLoading }).then(() => {
+      DataImport.uploadFile('marcFileForC350901.mrc', fileName);
+      JobProfiles.waitLoadingList();
+      JobProfiles.searchJobProfileForImport(jobProfileToRun);
+      JobProfiles.runImportFile();
+      JobProfiles.waitFileIsImported(fileName);
+      Logs.checkStatusOfJobProfile('Completed');
+      Logs.openFileDetails(fileName);
+      Logs.getCreatedItemsID().then(link => {
+        createdAuthorityID = link.split('/')[5];
+      });
+    });
+  });
+
+  beforeEach('', () => {
```

Comment:
Why you need 'beforeEach' block? you have only 1 it block

Reply:
for future tests in this file